Marlin/Marlin_main.cpp

```diff
@@ -1644,7 +1639,7 @@ static void clean_up_after_endstop_or_probe_move() {
 
     #if ENABLED(DELTA)
 
-      feedrate = XY_PROBE_FEEDRATE;
+      feedrate = (feed_rate != 0.0) ? feed_rate : XY_PROBE_FEEDRATE;
```

Comment:
I noticed a few places where `feedrate` was being set to `0.0`. Do we need to look closer at those?